Makefile.am

```diff
@@ -293,7 +293,7 @@ printdeps:
 # This is kind of a hack, but I couldn't find a better way to adjust the paths
 # in the script before it gets installed...
 install-exec-hook:
-	script=tsdb; pkgdatadir='$(pkgdatadir)'; configdir='$(sysconfigdir)/etc/opentsdb'; \
+	script=tsdb; pkgdatadir='$(pkgdatadir)'; configdir='$(prefix)/etc/opentsdb'; \
```

Comment:
My understading is that:
```
$(sysconfig)/opentsdb
```

should work on GNU/Linux too but unfortunately I have no Linux boxes to test this.